test/verify/check-users-roles

```diff
@@ -103,10 +103,12 @@ class TestRoles(MachineCase):
 
         self.allow_restart_journal_messages()
 
+    @nondestructive
     def testDynamic(self):
         m = self.machine
         b = self.browser
 
+        self.addCleanup(m.execute, "groupdel docker || true")
```

Comment:
We shouldn't need this, testlib.py already restores passwd/group and so on. But of course it doesn't really hurt.